src/js/messaging/containers/Compose.jsx

```diff
@@ -46,7 +47,11 @@ class Compose extends React.Component {
           <strong>Note:</strong> Messages may be saved to your health record at
           your team's discretion.
         </p>
-
+        <MessageFrom
+            cssClass="messaging-from"
+            lastName="Veteran"
+            firstName="Jane"
+            middleName="Q."/>
```

Comment:
Placeholder name because we don't yet have the ability to retrieve a veteran's name.